src/test/external/ext_out_cluster_test.go

```diff
@@ -136,6 +146,103 @@ func (suite *ExtOutClusterTestSuite) Test_1_Deploy() {
 	suite.True(success, errorStr)
 }
 
+func (suite *ExtOutClusterTestSuite) Test_2_AuthToPrivateHelmChart() {
+	baseURL := fmt.Sprintf("http://%s", "gitea.localhost:3000")
+
+	createHelmChartInGitea(suite, baseURL, giteaUser, giteaPassword)
+	makeUserPrivate(suite, baseURL, giteaUser, giteaPassword)
+
```

Comment:
instead of passing in `suite`, you can make these functions receivers on `suite`, just don't prefix them w/ `Test` and they will not be executed as tests by the suite itself